### PR TITLE
Mark deletions as - and mask <10x coverage as N

### PR DESCRIPTION
**Pull request by christinehe**

### CODE REVIEW

File: covid19_call_variants.sh [hunk 1]
```diff
@@ -108,8 +108,20 @@ bcftools view \
...
 bcftools index "${prefix}.vcf.gz"
 
+# We want to generate a consensus sequence with:
+# 1. well-supported deletions marked with '-'
+# 2. low-coverage regions masked with N
+
+# vcf is 1-based while bedgraph is 0-based; thus the somewhat convoluted path to mask.bed
+bcftools query -f'%CHROM\t%POS0\t%END\n' ${prefix}.vcf.gz > variants.bed
+# get low coverage sites (<10) in bedgraph format
+bedtools genomecov -bga -ibam ${prefix}.sorted.bam | awk '$4 < 10' > low_coverage_sites.bed
```
Comment (safay) [inline covid19_call_variants.sh:118]: @christinehe What do you think about exposing the value that defines low coverage as a parameter?

Comment (audy) [reply covid19_call_variants.sh:118]: +1 this will make it easy to generate a `consensus.fasta` at a bunch of different min depths.

Comment (christinehe) [reply covid19_call_variants.sh]: Resolved in [e2754e6](https://github.com/onecodex/sars-cov-2/pull/57/commits/e2754e6893a8bc80af15dc425ba248a7e90d0dc9)

File: jobscript.sh [hunk 2]
```diff
@@ -97,4 +99,26 @@ echo "Generating notebook!"
 #shellcheck disable=SC1000-SC9999
 RESULTS_DIR="$(pwd)" SAMPLE_PATH="${sample_filename}" PYTHONWARNINGS="ignore" GIT_DIR="/.git" GIT_WORK_TREE="/" INSTRUMENT_VENDOR="${INSTRUMENT_VENDOR}" ONE_CODEX_REPORT_FILENAME="${ONE_CODEX_REPORT_FILENAME}" ARTIC_PRIMER_VERSION="${ARTIC_PRIMER_VERSION}" conda run -n jobscript-env jupyter nbconvert --execute --to onecodex_pdf --ExecutePreprocessor.timeout=-1 --output="${ONE_CODEX_REPORT_FILENAME}" --output-dir="." report.ipynb
 
+echo "Removing unnecessary files"
+rm -f aa_codes.txt \
+	annot_table.orfs.txt \
+	covid19.bam.bai \
```
Comment (safay) [inline jobscript.sh:105]: Please add the index back in. It's easy to generate, true, but keeping it eliminates an extra step if someone wants to look at the data in IGV, and not all users will be immediately familiar with how to generate an index.

**Comment by safay** [comment]
@christinehe Just one minor quibble with the deletion of `.bai`, and get the tests to pass, and that's good enough for me.

Looks like you might need to pass in the depth variable to get the tests to pass; I believe this also means we'll need to update the Orbiter task, right?

**Pull request merged**